rsts/deployment/cluster_config/auth_setup.rst

```diff
@@ -228,12 +233,30 @@ To set up an external OAuth2 Authorization Server, follow the instructions below
 
 .. tabbed:: Okta
 
-   1. Under security -> API, click `Add Authorization Server`. Set the audience to the public URL of FlyteAdmin (e.g. https://flyte.mycompany.io/).
-   2. Under `Access Policies`, click `Add New Access Policy` and walk through the wizard to allow access to the authorization server.
-   3. Under `Scopes`, click `Add Scope`. Set the name to `all` (required) and check `Require user consent for this scope` (recommended).
-   4. Create 2 apps (for Flytectl and Flytepropeller) to enable these clients to communicate with the service.
-      Flytectl should be created as a `native client`.
-      Flytepropeller should be created as an `OAuth Service` and note the client ID and client Secrets provided.
+   Okta's custom authorization servers are available through an add-on license. The free developer accounts do include access, which you can use to test before rolling out the configuration more broadly.
+
+   1. Under security -> API, click `Add Authorization Server`. Set the audience to the public URL of FlyteAdmin (e.g. https://example.foobar.com).
```

Comment:
I have updated the example public URLs throughout this doc to `https://example.foobar.com` to match the first given example